examples/scripts/3_Dynamics/3.10_Hybrid_swap_mc.py

```diff
@@ -77,28 +72,31 @@ class HybridSwapMCState(MDState):
 
     last_permutation: torch.Tensor
     _atom_attributes = (
-        MDState._atom_attributes | {"last_permutation"}  # noqa: SLF001
+        ts.SwapMCState._atom_attributes | {"last_permutation"}  # noqa: SLF001
     )
 
 
-nvt_init, nvt_step = nvt_langevin(model=model, dt=0.002, kT=kT)
-md_state = nvt_init(state, seed=42)
+md_state = ts.nvt_langevin_init(model=model, state=state, kT=torch.tensor(kT), seed=42)
 
-swap_init, swap_step = swap_monte_carlo(model=model, kT=kT, seed=42)
-swap_state = swap_init(md_state)
+swap_state = ts.swap_mc_init(model=model, state=md_state)
 hybrid_state = HybridSwapMCState(
     **vars(md_state),
-    last_permutation=torch.zeros(
-        md_state.n_systems, device=md_state.device, dtype=torch.bool
+    last_permutation=torch.arange(
+        md_state.n_atoms, device=md_state.device, dtype=torch.long
     ),
 )
 
 generator = torch.Generator(device=device)
 generator.manual_seed(42)
 
 n_steps = 100
+dt = torch.tensor(0.002)
 for step in range(n_steps):
     if step % 10 == 0:
-        hybrid_state = swap_step(hybrid_state, kT=torch.tensor(kT), generator=generator)
+        hybrid_state = ts.swap_mc_step(
+            model=model, state=hybrid_state, kT=kT, seed=42 + step
```

Comment:
why not keep passing the same generator? reseeding is bad for entropy vs using the same prng sequence